dist/jquery.flot.js

```diff
@@ -3240,7 +3240,7 @@ Licensed under the MIT license.
                         // data units, because the scales of the axes may be different
                         var dx = Math.abs(axisx.p2c(x) - mouseX),
                             dy = Math.abs(axisy.p2c(y) - mouseY),
-                            dist = dx * dx + dy * dy; // we save the sqrt
+                            dist = dx * dx + dy * dy * 0.025; // we save the sqrt
```

Comment:
are you sure about this ? this is also used bu hover and is a public API. I wouldn't change it in this way.